### PR TITLE
Add hasTopSpacing support to AlertBannerService

### PR DESCRIPTION
**Pull request by DamienBurkeWN**
## Testing
1. View AlertBannerService and try out the hasTopSpacing control
2. Check Summary is still OK and uses darker blue in West theme

### CODE REVIEW

File: src/library/pages/CouncilTaxPageExample/CouncilTaxPageExample.tsx [hunk 1]
```diff
@@ -57,7 +57,6 @@ export const CouncilTaxPageExample: React.FC<CouncilTaxPageExampleProps> = ({ver
   <>
     <PageStructures.Header />
     <PageStructures.MaxWidthContainer>
-      <PageStructures.PhaseBanner />
```
Comment (DamienBurkeWN) [inline src/library/pages/CouncilTaxPageExample/CouncilTaxPageExample.tsx:60]: Removing the PhaseBanner from examples too

File: src/library/pages/SignpostPageExample/SignpostPageExample.tsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import * as PageStructures from '../../structure/PageStructures';
 import Heading from '../../components/Heading/Heading';
-import { hrefTo } from '@storybook/addon-links';
```
Comment (DamienBurkeWN) [inline src/library/pages/SignpostPageExample/SignpostPageExample.tsx:4]: Unused and dodgy package

File: src/library/structure/AlertBannerService/AlertBannerService.styles.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ export const Container = styled.div`
   margin-bottom: 25px;
   display: flex;
   justify-content: center;
+  margin-top: ${(props) => (props.hasTopSpacing ? `25px` : `0`)};
```
Comment (DamienBurkeWN) [inline src/library/structure/AlertBannerService/AlertBannerService.styles.js:21]: Actual meat of the change

File: src/library/structure/Summary/Summary.styles.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ const summaryStyles = (props) => {
       return css`
         background-color: ${(props) =>
           props.hasBackground ? props.theme.theme_vars.colours.grey_light : 'transparent'};
-        color: ${(props) => props.theme.theme_vars.colours.action};
+        color: ${(props) => props.theme.theme_vars.colours.action_dark};
```
Comment (DamienBurkeWN) [inline src/library/structure/Summary/Summary.styles.js:21]: Bonus accessibility contrast improvement (see Slack)

File: src/library/structure/AlertBannerService/AlertBannerService.styles.js (hunk 3 above)
Comment (chrisrhymes) [reply src/library/structure/AlertBannerService/AlertBannerService.styles.js:21]: So this will be for when a page has a hero image and a service alert?

Comment (DamienBurkeWN) [reply src/library/structure/AlertBannerService/AlertBannerService.styles.js:21]: Yep, just so

**Review by chrisrhymes — APPROVED** [review]
Good way of solving the issue. Good catch on the accessibility colour contrast too!

**Pull request merged**